emerald-lang-spec/4predicates.em

```diff
@@ -0,0 +1,5 @@
+(null? a)
+; compiles to: a.nil?
+
+(empty? (3 4))
+; compiles to: [3, 4].empty?
```

Comment:
Humm. I think we might need to think a little more about how we compile functions as later it's going to get a lot more complex as the user will be able to define their own.